core/db/migrate/20150213094131_attach_shipments_to_order_address.rb

```diff
@@ -0,0 +1,10 @@
+class AttachShipmentsToOrderAddress < ActiveRecord::Migration
+  def up
+    Spree::Shipment.where(address: nil).each do |shipment|
```

Comment:
It would be play nicer for larger stores if we used update_all here, so something like:

`Shipment.where(address:nil).joins(:order).update_all("shipments.address_id = orders.ship_address_id")`